src/WebUI/locales/en.yml

```diff
@@ -4,9 +4,10 @@ locale:
   cn: 中文
 
 platform:
-  Steam: Steam Workshop
+  Steam: Steam
   EpicGames: Epic Games
   Microsoft: Xbox
+  SteamWorkshop: Steam Workshop
```

Comment:
platform.* is the translation for the platform entity. Steam workshop is not a platform for user authorization.
You can simply create a translation in the root of the dictionary or not create it at all, like Discord for example

https://github.com/namidaka/crpg/blob/master/src/WebUI/src/models/platform.ts

Reply:
https://github.com/namidaka/crpg/pull/329/commits/9367cafae60a6be57a8dae8b6af969c2097c7183